### ISSUE TITLE
barWidth option broken after commit 18cad42e

### ISSUE THREAD

**Issue by sandiz** [katspaugh/wavesurfer.js#1628]
### Wavesurfer.js version(s):
latest master

### Browser and operating system version(s):
electron: 4.0.4 chrome: 69.0

hi! firstly thank you for the great library, i'm having a lot of fun working with it.

i have been keeping up with the latest master and noticed that the waveform has stopped generating with commit 18cad42e (canvasentry refactor)

pre 18cad42e:
![image](https://user-images.githubusercontent.com/5438703/56454736-45009680-630a-11e9-9c95-0187e136908c.png)

post 18cad42e:
![image](https://user-images.githubusercontent.com/5438703/56454751-75e0cb80-630a-11e9-97f0-7077fe0967fd.png)

my params are:
```javascript
const params = {
            audioContext: this.audioContext,
            container: '#waveform',
            waveColor: '#04ABED',
            progressColor: 'black',
            cursorColor: '#fff',
            barWidth: 3,
            barGap: 2,
            barRadius: true,
            height: 180,
            barHeight: 1,
            scrollParent: true,
            responsive: true,
            closeAudioContext: true,
            //forceDecode: true,
            plugins: [
                TimelinePlugin.create({
                    container: '#timeline',
                    primaryColor: "#fff",
                    fontFamily: 'Roboto Condensed',
                }),
                MinimapPlugin.create({
                    container: "#minimap",
                    waveColor: '#000',
                    progressColor: 'black',
                    height: 30,
                    showOverview: true,
                    barWidth: 0,
                    barGap: 0,
                    barRadius: false,
                    overviewBorderColor: "azure",
                }), 
            ],
        };
```

let me know if you need more info!

**Comment by thijstriemstra** [comment]
thanks for the report @sandiz. can you also try to narrow it down (browser-only, not electron and disable plugins for example)?

**Comment by sandiz** [comment]
hi @thijstriemstra, tried narrowing it down by using the spectogram example. it seems `barWidth` is the culprit..

with latest master and barWidth:

![image](https://user-images.githubusercontent.com/5438703/56536852-ee66a880-6513-11e9-839f-e2a87c046238.png)


with latest master and barWidth commented: 

![image](https://user-images.githubusercontent.com/5438703/56536804-cd05bc80-6513-11e9-8b20-43f32a1d277d.png)


my params:
```
document.addEventListener('DOMContentLoaded', function () {
    // Create an instance
    var options = {
        container: '#waveform',
        waveColor: 'blue',
        progressColor: 'black',
        cursorColor: '#fff',
        height: 180,
        scrollParent: true,
        responsive: true,
        closeAudioContext: true,
        barGap: 2,
        barHeight: 1,
        //barWidth: 3,
        plugins: [
            WaveSurfer.spectrogram.create({
                container: '#wave-spectrogram',
                labels: true
            })
        ]
    };
```

let me know if you need more info! i can investigate further if you give me some pointers..

**Comment by sandiz** [comment]
the original waveform in my electron app also starts working if i comment out barWidth
![image](https://user-images.githubusercontent.com/5438703/56537104-911f2700-6514-11e9-89da-096b408322ef.png)

**Comment by thijstriemstra** [comment]
I didn't properly test barWidth option while working on the `CanvasEntry` code, issue is somewhere in change made to `MultiCanvas` here: https://github.com/katspaugh/wavesurfer.js/compare/2.2.1...master#diff-b69b942b0f75bc91abe0a1690a528fb8

**Comment by thijstriemstra** [comment]
> i can investigate further if you give me some pointers..

@sandiz if you can help figure out what broke during that refactor, much appreciated, I'm focusing on other areas right now. This issue does block the 3.0.0 release though, which sucks.

**Comment by sandiz** [comment]
hi @thijstriemstra 
pushed a fix for this in pr #1637, some arguments were missing.
